### PR TITLE
Fix #94 Refresh guest info before check ip address

### LINKED ISSUE #94: [Bug] [Intermittent] check_ip_address failed occasionally
##### SUMMARY

```
2021-07-08 13:07:34,008 | Failed at Play [check_ip_address] **************************

2021-07-08 13:07:34,008 | TASK [Check IP address lists difference] *******************
task path: /root/ansible-vsphere-gos-validation/linux/check_ip_address/validate_ip_address.yml:17
fatal: [localhost]: FAILED! => {
    "assertion": "all_ip_in_guest | difference(all_ip_by_tools) | length == 0",
    "changed": false,
    "evaluated_to": false,
    "msg": "\"VM IP addresses got in guest info: ['10.193.8.193', 'fe80::250:56ff:feb5:6db2'], expected IP addresses got in guest OS: ['10.193.8.193', 'fd01:1:2:2918:250:56ff:feb5:6db2', 'fe80::250:56ff:feb5:6db2']\"\n"
}
```
Rerun could pass. 

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below
ansible-playbook [core 2.11.1] 
  config file = /root/ansible-vsphere-gos-validation/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible-playbook
  python version = 3.9.1 (default, May 19 2021, 10:27:03) [GCC 10.2.0]
  jinja version = 3.0.1
  libyaml = True


```

##### CONFIGURATION

```paste below



```

##### ENVIRONMENT / OS

RHEL 8.4

### PR DESCRIPTION
**Pull request by keirazhang**
Signed-off-by: Qi Zhang <qiz@vmware.com>

### CODE REVIEW

**Comment by keirazhang** [comment]
```
+-----------------------------------------------------------------------------------+
| Guest OS Type             | RedHat 9.0 x86_64                                     |
+-----------------------------------------------------------------------------------+
| VMTools Version           | 11.3.0.29534 (build-18090558)                         |
+-----------------------------------------------------------------------------------+
| CloudInit Version         |                                                       |
+-----------------------------------------------------------------------------------+
| Config Guest Id           | rhel9_64Guest                                         |
+-----------------------------------------------------------------------------------+
| Hardware Version          | vmx-20                                                |
+-----------------------------------------------------------------------------------+
| GuestInfo Guest Id        | rhel9_64Guest                                         |
+-----------------------------------------------------------------------------------+
| GuestInfo Guest Full Name | Red Hat Enterprise Linux 9 (64-bit)                   |
+-----------------------------------------------------------------------------------+
| GuestInfo Guest Family    | linuxGuest                                            |
+-----------------------------------------------------------------------------------+
| GuestInfo Detailed Data   | architecture='X86'                                    |
|                           | bitness='64'                                          |
|                           | distroName='Red Hat Enterprise Linux'                 |
|                           | distroVersion='9.0'                                   |
|                           | familyName='Linux'                                    |
|                           | kernelVersion='5.14.0-1.7.1.el9.x86_64'               |
|                           | prettyName='Red Hat Enterprise Linux 9.0 Beta (Plow)' |
+-----------------------------------------------------------------------------------+

Test Results (Total: 3, Failed: 0, No Run: 0, Elapsed Time: 00:18:53):
+-----------------------------------------+
| Name               | Status | Exec Time |
+-----------------------------------------+
| ovt_verify_install | Passed | 00:12:05  |
| ovt_verify_status  | Passed | 00:04:25  |
| check_ip_address   | Passed | 00:01:27  |
+-----------------------------------------+
```

File: linux/check_ip_address/vm_wait_guest_all_ip.yml [hunk 1]
```diff
@@ -1,20 +1,62 @@
...
+  register: vm_guestinfo_net
+  until:
+    - vm_guestinfo_net is defined
+    - vm_guestinfo_net.instance is defined
+    - vm_guestinfo_net.instance.guest is defined
+    - vm_guestinfo_net.instance.guest.net is defined
+    - all_ip_in_guest | difference(vm_guestinfo_net.instance.guest.net | map(attribute='ipAddress') | flatten) | length == 0
+  retries: 10
+  delay: 5
+  ignore_errors: True
```
Comment (Tomorrow9) [inline linux/check_ip_address/vm_wait_guest_all_ip.yml:47]: If "all_ip_in_guest" and the IP addresses got in guest info are not the same, this task will not fail and the test case will not fail?

Comment (keirazhang) [reply linux/check_ip_address/vm_wait_guest_all_ip.yml:47]: The case will fail at line 58

Comment (Tomorrow9) [reply linux/check_ip_address/vm_wait_guest_all_ip.yml:47]: Ok.

**Review by Tomorrow9 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**